src/main/java/carRacing/model/RandomMoveStrategy.java

```diff
@@ -0,0 +1,18 @@
+package carRacing.model;
+
+public class RandomMoveStrategy implements MoveStrategy {
+
+    private static final int STANDARD_NUMBER = 4;
+    private final RandomGenerator randomGenerator;
+
+    public RandomMoveStrategy(RandomGenerator randomGenerator){
+
+        this.randomGenerator = randomGenerator;
```

Comment:
```suggestion
    public RandomMoveStrategy(RandomGenerator randomGenerator){
        this.randomGenerator = randomGenerator;
```
불필요한 공백 라인은 없어도 되지 않을까?